src/workloads/docs/CrudActorFSM.yml

```diff
@@ -153,5 +153,5 @@ Actors:
 
     # In this example we always start in the off State
     InitialStates:
-    - "Off":
-        Weight: 1
+    - State: "Off"
```

Comment:
Update to the syntax example. Encoding values in keys is hard, so I moved the state name into a proper field.